app/src/main/res/values/strings.xml

```diff
@@ -49,6 +49,7 @@
     <string name="sender_and_message">Sender and message</string>
     <string name="sender_only">Sender only</string>
     <string name="enable_delivery_reports">Enable delivery reports</string>
+    <string name="use_simple_characters">Use simple characters</string>
```

Comment:
This is a quite non descriptive string, replace it with "Remove accents and diacritics at sending messages" please